studio/.env.docker

```
@@ -12,4 +12,7 @@ NEXT_PUBLIC_SENTRY_DSN=APP_NEXT_PUBLIC_SENTRY_DSN
 NEXT_PUBLIC_SENTRY_CLIENT_SAMPLE_RATE=APP_NEXT_PUBLIC_SENTRY_CLIENT_SAMPLE_RATE
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SENTRY_CLIENT_SAMPLE_RATE key should go before the NEXT_PUBLIC_SENTRY_DSN key

(UnorderedKey)